python/dgl/data/chem/csv_dataset.py

```diff
@@ -90,8 +102,8 @@ def __getitem__(self, item):
             Weights of the datapoint for all tasks
         """
         return self.smiles[item], self.graphs[item], \
-               F.zerocopy_from_numpy(self.labels[item]),  \
-               F.zerocopy_from_numpy(self.mask[item])
+            F.zerocopy_from_numpy(self.labels[item]),  \
+            F.zerocopy_from_numpy(self.mask[item])
```

Comment:
Let not use backend API in data package. Also returning dgl ndarray to user is not friendly. Directly return numpy arrays?